server/src/main/java/org/apache/cassandra/sidecar/db/ConfigAccessorFactory.java

```diff
@@ -25,29 +27,22 @@
  */
 public class ConfigAccessorFactory
 {
-    private final KafkaConfigAccessor kafkaConfigAccessor;
-    private final CdcConfigAccessor cdcConfigAccessor;
+    private final Map<Service, ConfigAccessor> configAccessors = new HashMap<>();
 
     @Inject
     public ConfigAccessorFactory(KafkaConfigAccessor kafkaConfigAccessor,
                                  CdcConfigAccessor cdcConfigAccessor)
     {
-        this.kafkaConfigAccessor = kafkaConfigAccessor;
-        this.cdcConfigAccessor = cdcConfigAccessor;
+        configAccessors.put(Service.KAFKA, kafkaConfigAccessor);
+        configAccessors.put(Service.CDC, cdcConfigAccessor);
     }
 
-    public ConfigAccessor getConfigAccessor(final String service)
+    public ConfigAccessor getConfigAccessor(final Service service)
     {
-        if (service.equals(Service.KAFKA.serviceName))
+        if (configAccessors.containsKey(service))
         {
-            return kafkaConfigAccessor;
+            return configAccessors.get(service);
         }
-
-        if (service.equals(Service.CDC.serviceName))
-        {
-            return cdcConfigAccessor;
-        }
-
         throw new RuntimeException("Couldn't find a db accessor for service " + service);
     }
```

Comment:
prefer not containing `get` in the method name, remove `final` and look up just once.

```suggestion
    public ConfigAccessor configAccessor(Service service)
    {
        ConfigAccessor accessor = configAccessors.get(service);
        if (accessor == null)
        {
            throw new RuntimeException("Couldn't find a db accessor for service " + service);
        }
        
        return accessor; 
    }
```

Reply:
Is there any reason to not to do containsKey call? is this to avoid calling `get()` & `containsKey` twice? I feel that checking containsKey is cleaner than making a null check.